src/test/ui/did_you_mean/issue-38147-3.stderr

```diff
@@ -0,0 +1,13 @@
+error: cannot borrow immutable borrowed content `*self.s` as mutable
+  --> $DIR/issue-38147-3.rs:17:9
+   |
+12 |     s: &'a String
+   |     ------------- this field should be `&mut`
+...
+16 |     fn f(&self) {
+   |          ----- use `&mut self` here to make mutable
```

Comment:
maybe we can adjust the wording to make it clearer that BOTH parts need to be `&mut self`? That's a bit tricky, particularly since we don't control the relative order of things here